enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/SøknadType.kt

```diff
@@ -11,5 +11,13 @@ enum class SøknadType(val dokumentType: String, val dokumentTypeVedlegg: String
         fun hentSøknadTypeForDokumenttype(dokumentType: String): SøknadType {
             return dokumenttypeMap[dokumentType] ?: throw error("Fant ikke søknadtype for dokumenttype $dokumentType")
         }
+
+        private val søknadstyperMedDokumentasjonsbehov = listOf(BARNETILSYN,
```

Comment:
Spiller kanskje ingen veldig stor rolle for denne, men generellt fint å bruke set når man skal sjekke om en "liste" inneholder ett verdi :) 

Her hadde man også kunnet utvide SøknadType med en egen metode, eller verdi, for om den har dokumentasjonsverdi, eks `val harDokumentasjonsbehov: Boolean = false` i enum constructor

Reply:
Enig :) Gjorde endringen nå